test/functional/test_helper.py

```diff
@@ -0,0 +1,15 @@
+def get_available_port(in_range):
+    """Returns available local port number.
+    """
+    range(8102, 8200)
+    if not isinstance(in_range, range):
```

Comment:
in python3 range creates a generator by default. I would rather provide just start and end port numbers with default valuesof 8102 and 8200 as method arguments